array_method_lab.js

```diff
@@ -0,0 +1,182 @@
+console.log(`Question One`);
```

Comment:
Overall good use of array methods. I see that you understood how to use them and also how to implement anonymous callback functions for those methods.